versions.cfg

```diff
@@ -256,3 +256,8 @@ elasticsearch = 5.4.0
 
 # Added by buildout at 2018-01-17 17:26:37.230945
 selenium = 3.8.1
+
+# Required by:
+# encoded==66.0
+alembic = 0.9.8
+python-editor = 1.0.3
```

Comment:
What is the python-editor for?

Reply:
It's a dependency of `alembic`. It looks like a small utility to help with opening the user's preferred text editor from Python.

Reply:
Dependency of Alembic.